backend/src/v5/models/tickets.groups.js

```diff
@@ -37,15 +38,29 @@ Groups.getGroupsByIds = (teamspace, project, model, ticket, groupIds, projection
 	{ teamspace, project, model, ticket, _id: { $in: groupIds } }, projection);
 
 Groups.updateGroup = async (teamspace, project, model, ticket, groupId, data, author) => {
-	const updates = { $set: data };
+	const toUpdate = { };
+	const toUnset = { };
 
-	if (data.rules) {
-		updates.$unset = { objects: 1 };
-	} else if (data.objects) {
-		updates.$unset = { rules: 1 };
+	Object.keys(data).forEach((key) => {
+		if (data[key]) {
+			toUpdate[key] = data[key];
+		} else {
+			toUnset[key] = 1;
+		}
+	});
+
+	if (toUpdate.rules) {
+		toUnset.objects = 1;
+	} else if (toUpdate.objects) {
+		toUnset.rules = 1;
 	}
+
+	const actions = {};
+	if (!isEqual(toUpdate, {})) actions.$set = toUpdate;
```

Comment:
Probably more of a readibility thing here, but I think it's better to do `Object.keys(...).length`